lobpcg.jl

```diff
@@ -80,7 +82,7 @@ function ortho(X; tol=1e-10)
         # a good a posteriori error is that X'X - I is eps()*κ(R)^2;
         # in practice this seems to be sometimes very overconservative
         success && eps(real(eltype(X)))*condR^2 < tol && break
-        # println(norm(X'X - I), ", ", norm((X/R)'*(X/R) - I), ", ", cond(R)^2*eps())
+        vprintln(norm(X'X - I), ", ", norm((X/R)'*(X/R) - I), ", ", cond(R)^2*eps())
```

Comment:
Might as well remove that, it's not terribly interesting

Reply:
Ok, done.